tests/integration/components/flash-message-test.js

```diff
@@ -34,3 +40,72 @@ test('it does not error when quickly removed from the DOM', function(assert) {
 
   assert.ok(this.get('flash').isDestroyed, 'Flash Object isDestroyed');
 });
+
+test('flash message is removed after timeout', function(assert) {
+  assert.expect(3);
+  clock = sinon.useFakeTimers();
+
+  let destroyMessage = sinon.spy();
+
+  this.set('flash', FlashMessage.create({
+    message: 'hi',
+    sticky: false,
+    timeout: timeoutDefault,
+    destroyMessage
+  }));
+
+  this.render(hbs`
+    {{#flash-message flash=flash as |component flash|}}
+      {{flash.message}}
+    {{/flash-message}}
+  `);
+
+  assert.equal(this.$().text().trim(), 'hi');
+  assert.notOk(destroyMessage.calledOnce, 'flash has not been destroyed yet');
+
+  clock.tick(timeoutDefault);
+  assert.ok(destroyMessage.calledOnce, 'flash is destroyed after timeout');
+
+  clock.restore();
+});
+
+test('flash message is removed after timeout', function(assert) {
```

Comment:
Try running `ember try:one ember-1.13 --- ember test --serve` and see if there are any errors in the console